jarviscli/plugins/horoscope.py

```diff
@@ -0,0 +1,78 @@
+from plugin import plugin, require, alias
+import pyaztro
+
+@require(network=True)
+@alias("astro")
+@plugin("horoscope")
+class Horoscope :
+
+    def __call__(self, jarvis: "JarvisAPI", s: str) -> None:
+        self.print_horoscope(jarvis)
+
+    def print_horoscope(self, jarvis: "JarvisAPI") :
+        jarvis.say("Which astrological sign are you?")
+        jarvis.say("1: Aries")
+        jarvis.say("2: Taurus")
+        jarvis.say("3: Gemini")
+        jarvis.say("4: Cancer")
+        jarvis.say("5: Leo")
+        jarvis.say("6: Virgo")
+        jarvis.say("7: Libra")
+        jarvis.say("8: Scorpio")
+        jarvis.say("9: Sagittarius")
+        jarvis.say("10: Capricorn")
+        jarvis.say("11: Aquarius")
+        jarvis.say("12: Pisces")
+        sign = int(jarvis.input("Enter your number: "))
+        if(sign<1 or sign>12) :
+            jarvis.say("Wrong number input (Should be between 1 and 12). ")
+        else :
+            self.info_sign(jarvis, sign)
+
+    def info_sign(self, jarvis: "JarvisAPI",n : int) :
+        match n :
```

Comment:
Oh, match is new in python 3.10. Which is a nice feature and read great - but for backward-compatibility, a classic if-elsif-else would be better